Add announcement for week 48 (#542)
giantswarm/releases: announcements/2020-Week-48/announcement.md

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.

> 2. [Cert Manager v2.3.3](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#233---2020-11-23) schedules hook Jobs on master nodes.

Author: @giantswarm/app-squad-cert-manager can you help me - how does it make customer's life better (if any)? 😀

Reviewer: this one is an improvement :D it means customers can create AWS clusters with no nodepools and cert-manager will still start up correctly.

> 3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.

Author: @giantswarm/app-squad-nginx can you help me - how does it make customer's life better (if any)? 😀

Reviewer: no real customer-facing changes here, aside from the version bump which fixes some bugs.

Author: ah ok! better to not include it? wdyt?

Author: I will remove it.

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.

## Web UI

In [Happa](https://docs.giantswarm.io/changes/web-ui/), deleting an organization is now simpler and upgrading to beta versions is now possible.

## Documentation

> - The article on the [operational layers](https://docs.giantswarm.io/basics/giant-swarm-operational-layers/) got an overhaul.

Author: @marians in this section you prefer to link to the doc itself rather than https://docs.giantswarm.io/changes/documentation/docs/v1.18.8/?

Author: I think it is better. It's confusing when user clicks and see changelog. too many clicks to read the doc.

> - We added explanation for the use of deprecated tenant cluster releases where it matters.

Author: ```suggestion
- In [Cluster Upgrades with Giant Swarm](https://docs.giantswarm.io/reference/cluster-upgrades/), we added an explanation for the use of deprecated tenant cluster releases where it matters.
```

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.

## Web UI

> In [Happa](https://docs.giantswarm.io/changes/web-ui/), deleting an organization is now simpler and upgrading to beta versions is now possible.

Author: @marians I edited to remove versions.

> [Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplifies deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) adds support for upgrading to beta versions.

Author: ```suggestion
In [Happa](https://docs.giantswarm.io/changes/web-ui/), deleting an organization is now simpler and upgrading to beta versions is now possible.
```

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.

## Web UI

> 

Author: ```suggestion
```

> [Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplified deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) added support for upgrading to beta versions.

Author: ```suggestion
[Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplifies deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) adds support for upgrading to beta versions.
```

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.

> 2. [Cert Manager v2.3.3](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#233---2020-11-23) schedules hook Jobs on master nodes.

Author: ```suggestion
2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.
```

> 3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.

Author: ```suggestion
```

> 3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.

Author: ```suggestion
```

## Web UI

> Prior, [Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplified deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) added support for upgrading to beta versions.

Author: ```suggestion
[Happa v1.1.1](https://github.com/giantswarm/happa/releases/tag/v1.1.1) simplified deleting an organization and [Happa v1.1.0](https://github.com/giantswarm/happa/releases/tag/v1.1.0) added support for upgrading to beta versions.
```

## Documentation

> - We extended our guide on [automatic termination of unhealthy nodes](https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/) for Azure, where this function is available as tenant cluster release v13.1.0.

Reviewer 2: ```suggestion
```

This feature is not yet released

- The article on the [operational layers](https://docs.giantswarm.io/basics/giant-swarm-operational-layers/) got an overhaul.
- We added explanation for the use of deprecated tenant cluster releases where it matters.

> - The guide on [adding labels to tenant clusters](https://docs.giantswarm.io/guides/tenant-cluster-labelling/) has been updated to inform you that this is possible on Azure.

Reviewer 2: @axbarsan was this released?

Reviewer 3: I took these from docs release notes (https://docs.giantswarm.io/changes/documentation/), so the answer is yes

Reviewer 2: > or v{{% first_azure_nodepools_version %}} and above for Azure

First azure nodepools version is v13.0.0 and it was not released yet

Reviewer 3: Which docs page is that?

Reviewer 3: This is highly confusing, because https://docs.giantswarm.io/guides/tenant-cluster-labelling/ does not mention Azure at all

Reviewer 2: yes, exactly. 

I got it from Adrian's PR: https://github.com/giantswarm/docs/pull/667/files it was merged, I guess it wasn't released yet?

Reviewer 3: I removed the labelling item

Reviewer 2: Anyway, can we just remove it from the announcement to block it?

# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:

For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.

## Managed Apps

1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
2. [Cert Manager v2.3.3](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#233---2020-11-23) schedules hook Jobs on master nodes.
3. [NGINX IC v1.11.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1110---2020-11-18) updates image to v0.41.2 and adds ability to extend the app with specific values from appcatalog.

## Web UI

> [Happa v1.1.2](https://github.com/giantswarm/happa/releases/tag/untagged-76e708b4d31d651bb3db) adds support for selecting no availabilty zones for Azure masters.

Author: ```suggestion
```
This is still in draft. So removing.
